### ISSUE TITLE
Incorporate help-text describing each content type

### ISSUE THREAD

**Issue by monika-aashe** [AASHE/hub#155]
We would like to incorporate help-text for the content types, so that folks can get a better sense of which form is most appropriate when submitting. 

I'm open to how this will look in terms of design and functionality (hover-over versus in-line text, etc.)

**Comment by monika-aashe** [comment]
Checking with Julian and Chris P. to see whether they think it's a good idea for this to show up on the Content Type landing page as well as the Submit a Resource page, or just the latter.

**Comment by jamstooks** [comment]
Let's talk about this on our call tomorrow... just want to know the difference between introduction and the blurb.

**Comment by jamstooks** [comment]
So I'm not super happy with the way the [Discover page-style](http://htmlstream.com/preview/unify-v1.9/page_home8.html) works here. It feels really cluttered.

![screen shot 2016-01-19 at 3 49 55 pm](https://cloud.githubusercontent.com/assets/585436/12434524/88323118-bec4-11e5-9cb5-4b270d932a65.png)

**Comment by jamstooks** [comment]
That text just doesn't feel that necessary.

**Comment by monika-aashe** [comment]
Sounds good, it's on the agenda for tomorrow. I do like that this version
has more white space. We could maybe go back to hover-over text too.

For some reason, I get an optical illusion in the current version where my
eye is drawn to the white space between the corners (is anyone else seeing
a grey circle at each corner?)

On Tue, Jan 19, 2016 at 5:55 PM, Benjamin W Stookey <
notifications@github.com> wrote:

> That text just doesn't feel that necessary.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AASHE/hub/issues/155#issuecomment-173014730>.
>



-- 
Monika Urbanski
Data & Content Manager
Association for the Advancement of Sustainability in Higher Education
www.aashe.org | monika.urbanski@aashe.org | (888) 347-9997 ext. 26

-- 
Join us in Baltimore, Maryland from Oct. 9 - 12 for the AASHE 2016 
Conference & Expo, themed Beyond the Campus. Call for proposals and 
reviewers now open <https://ww4.aievolution.com/ash1601>!

Connect with us:   Website <http://www.aashe.org/> | Facebook 
<https://www.facebook.com/aasheorg> | Twitter 
<https://twitter.com/AASHENews> | LinkedIn 
<http://www.linkedin.com/company/aashe>

------------------------------
Email Confidentiality Notice 
<http://www.aashe.org/email_confidentiality_notice>

**Comment by monika-aashe** [comment]
I have a version in the ole mock-up that shows the category titles, which someone from the review team pointed out would be useful in helping to explain the varying colors: 
<img width="531" alt="screen shot 2016-01-19 at 6 00 40 pm" src="https://cloud.githubusercontent.com/assets/15214797/12434721/9842ffc6-bed6-11e5-9555-1306f7317992.png">

**Comment by peltonc** [comment]
Yes - I'm glad it's not just me! The rounded corners of the boxes create
grey circles in my peripheral vision too. It's minor maybe, but still
pretty annoying.

I think I would like the Discover page style if we were to get rid of the
descriptive text.


On Tue, Jan 19, 2016 at 10:59 PM, monika-aashe <notifications@github.com>
wrote:

> Sounds good, it's on the agenda for tomorrow. I do like that this version
> has more white space. We could maybe go back to hover-over text too.
>
> For some reason, I get an optical illusion in the current version where my
> eye is drawn to the white space between the corners (is anyone else seeing
> a grey circle at each corner?)
>
> On Tue, Jan 19, 2016 at 5:55 PM, Benjamin W Stookey <
> notifications@github.com> wrote:
>
> > That text just doesn't feel that necessary.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/AASHE/hub/issues/155#issuecomment-173014730>.
> >
>
>
>
> --
> Monika Urbanski
> Data & Content Manager
> Association for the Advancement of Sustainability in Higher Education
> www.aashe.org | monika.urbanski@aashe.org | (888) 347-9997 ext. 26
>
> --
> Join us in Baltimore, Maryland from Oct. 9 - 12 for the AASHE 2016
> Conference & Expo, themed Beyond the Campus. Call for proposals and
> reviewers now open <https://ww4.aievolution.com/ash1601>!
>
> Connect with us: Website <http://www.aashe.org/> | Facebook
> <https://www.facebook.com/aasheorg> | Twitter
> <https://twitter.com/AASHENews> | LinkedIn
> <http://www.linkedin.com/company/aashe>
>
> ------------------------------
> Email Confidentiality Notice
> <http://www.aashe.org/email_confidentiality_notice>
>
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AASHE/hub/issues/155#issuecomment-173015604>.
>

-- 
Join us in Baltimore, Maryland from Oct. 9 - 12 for the AASHE 2016 
Conference & Expo, themed Beyond the Campus. Call for proposals and 
reviewers now open <https://ww4.aievolution.com/ash1601>!

Connect with us:   Website <http://www.aashe.org/> | Facebook 
<https://www.facebook.com/aasheorg> | Twitter 
<https://twitter.com/AASHENews> | LinkedIn 
<http://www.linkedin.com/company/aashe>

------------------------------
Email Confidentiality Notice 
<http://www.aashe.org/email_confidentiality_notice>

**Comment by monika-aashe** [comment]
We decided to go with Discover page style, 3-column format but without the descriptive text. Hold off on adding the four category names for now. Ben will work on a content type layout (may need to be a different style than the topic landing page).

**Comment by monika-aashe** [comment]
Ben will upload screenshots of the desired layout - can share with review team and collect other suggestions.

**Comment by monika-aashe** [comment]
@jamstooks - can you share a screenshot of the 3-column design with the circular buttons without the added text? We will include this in the Review Team discussion when we have it to see what direction we can agree on.

**Comment by monika-aashe** [comment]
Okay, I think I was able to patch something together so you don't have to find this anymore @jamstooks 
![design3 screen shot](https://cloud.githubusercontent.com/assets/15214797/13182516/24ac2866-d700-11e5-9087-f96a0bcdc60d.png)

**Comment by monika-aashe** [comment]
Review team decision: stick with larger rectangles, but maybe make them smaller. Okay if this is a post launch improvement.
